src/components/InputGlobal/index.tsx

```diff
@@ -70,11 +69,15 @@ const Input: React.FC<InputGlobal> = ({
 
   const handleTooltipFocus = (e: any) => {
     if (!(e.target.id === 'PrePageTooltip' || e.target.id === 'SearchIcon')) {
-      if (inputRef.current !== null) inputRef.current.value = '';
+      // if (inputRef.current !== null) inputRef.current.value = '';
       setShowTooltip(false);
     }
   };
 
+  const handleClick = () => {
+    setShowTooltip(true);
```

Comment:
should not set tooltip to true just by clicking in input box, makes no sense:
![image](https://github.com/klever-io/kleverscan/assets/63923113/a348220b-eefa-42ab-b0f3-c3a986e0b8b3)